### DISCUSSION TITLE
Auto-suggest existing selector names on Selector Manager

### DISCUSSION THREAD

**Discussion by gnbrndn** [GrapesJS/grapesjs#3890] [Ideas]
#### What are you trying to add to GrapesJS?

Auto-suggest existing selectors on selector manager.

#### Describe your feature request detailed

Auto-suggest on selector manager. Tried tweaking parts of Selector Manager view to add [data-list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist), then referenced it to  `data-input` and add a method to fetch all selectors from `SelectorManager`.

<img width="421" alt="Screen Shot 2021-10-25 at 9 14 56 PM" src="https://user-images.githubusercontent.com/29661216/138703073-a68b2059-331c-405a-9e30-cab75d006491.png">

I can open an pull request for this for anyone to check. Related PR https://github.com/artf/grapesjs/pull/3900
Thanks!

**Comment by artf** [comment]
Hi @gnbrndn thank you for the PR but unfortunately, I have to decline it.
I don't want to push more UI-related stuff inside the core. The goal I'm trying to reach now is to make the core more "headless" and API-first, which should allow easier UI customization. Indeed in the last version, I've improved the Selector Manager with more API which should allow the [customization](https://grapesjs.com/docs/modules/Selectors.html#customization) without being locked by GrapesJS UI choices.
